handlebars-cmd/plan.sh

```diff
@@ -0,0 +1,62 @@
+pkg_name=handlebars-cmd
+pkg_origin=core
+pkg_version="0.1.4"
+pkg_maintainer="The Habitat Maintainers <humans@habitat.sh>"
+pkg_license=('MIT')
+pkg_source=nosuchfile.tar.gz
+pkg_deps=(
+  core/coreutils
+  core/node
+)
+pkg_build_deps=(
+  core/coreutils
+  core/node
+)
+pkg_description="handlebars command-line tool. Shrinkwrapped with handlebars 4.0.6."
+pkg_upstream_url="https://github.com/spion/handlebars-cmd"
+pkg_bin_dirs=(bin)
+pkg_interpreters=(bin/env)
```

Comment:
This isn't necessary. You only need `pkg_interpreters` if your package is shipping something somebody would use in a shebang.